src/tests/unit/utils/formatTime.test.js

```diff
@@ -10,7 +10,7 @@ describe('formatUtils.js', () => {
 
     it('returns an empty string for an invalid date', () => {
```

Comment:
Is this an invalid date, still? It appears to be valid?

Reply:
forgot to update description! thanks Matt hehe